Documentation/PageTsconfig/Mod/_WebList/_downloadPresets.tsconfig

```diff
@@ -0,0 +1,14 @@
+mod.web_list.downloadPresets {
+    pages {
+        10 {
+            label = Quick overview
+            columns = uid, title, crdate, slug
+        }
+
+        20 {
+            identifier = LLL:EXT:myext/Resources/Private/Language/locallang.xlf:preset2.label
+            label = UID and titles only
+            columns = uid, title
```

Comment:
The identifier is used as the `<option>` value-attribute. IMO no need to translate that.

```suggestion
            identifier = uid-title
            label = LLL:EXT:myext/Resources/Private/Language/locallang.xlf:preset2.label
            columns = uid, title
```